app/scripts/lib/oauth-client.js

```diff
@@ -39,8 +39,6 @@ function (xhr, OAuthErrors) {
 
     // params = { assertion, client_id, scope }
     getToken: function (params) {
-      /* jshint camelcase: false */
-
```

Comment:
@TDA: Let's add `camelcase: 2` to the base /.eslintrc file and tweak the 19 places where we have errors and disable the rule on those specific lines.